www/content/events.md

```diff
@@ -192,7 +192,7 @@ document.body.addEventListener('htmx:confirm', function(evt) {
     evt.preventDefault();
     swal({
       title: "Are you sure?",
-      text: "Are you sure you are sure?",
+      text: evt.detail.question,
```

Comment:
Not sure this is relevant here, as [the doc](https://htmx.org/docs/#confirming-requests-using-events) states:
> This event is fired on every trigger for a request (not just on elements that have a hx-confirm attribute)

So in the context of this doc, we can't assume that a `hx-confirm` attribute is set, and it likely is not since this example is precisely about using another confirm mechanism from a lib, instead of htmx's one